rfcs/rfc-external-project-and-gem-docs.md

```diff
@@ -0,0 +1,54 @@
+# Establish process for external project and Gem documentation
+
+## Summary
+
+This RFC proposes to clarify whether documentation about external projects and Gems can be included in O3DE Documentation (`o3de.org/docs`). For the context of this RFC, "external" refers to those that live outside of the `o3de` repository.
+
+External projects and Gems commonly fall under the following scenarios:
+- They live outside of the `o3de` organization, such as those created by O3DE partners.
+- Their repositories are part of the `o3de` organization, but not part of the `o3de` repository.
+
+Examples include docs for templates and Gems in `o3de-extras` repo, and tutorials for the Multiplayer Sample Project. 
+
+### What is the motivation for this suggestion?
+
+As the collection of O3DE projects and Gems expands, we need to establish boundaries about what documentation can be accepted into O3DE Documentation. It's important to set this boundary because it can directly impact the scope of responsibilities for D&C SIG and other O3DE maintainers.
+
+A current case is documentation for the Multiplayer Sample Project (MSP). While MSP's code base lives in a separate repository within `o3de` organization, its documentation is planned to live within O3DE Documentation. This decision was based on MSP's impact on the O3DE community, and consensus among D&C SIG and other O3DE community members who are stakeholders of MSP. 
+
+
+### Suggestion design description
+
+We propose setting the following boundaries for external projects and Gems:
+
+> Documentation for external projects and Gems that live outside of the `o3de` organization cannot be contained in O3DE Documentation. They must be hosted elsewhere by the owners.
+>
+> Documentation for external projects and Gems that are part of the `o3de` organization *may be considered* to become part of O3DE Documentation. The consideration will be dependent on the project or Gem's purpose and beneficial impact on the O3DE community. Discussions can be held with the D&C SIG and other SIG or TSC members who are stakeholders for the project or Gem in question, in order to make a decision.
+>
+> Although not all project and Gem documentation may be included in O3DE Documentation, a reference to them can be listed in the O3DE Documentation. For example, PopocornFX's particle effects is listed in the "Rendering" page of the O3DE Documentation.
```

Comment:
We've talked about having a centralized reference topic with a table of descriptions and links to external Gems and tutorials. What do you think of the following? If this is acceptable, it should probably be added to the adoption strategy section, below.
```suggestion
> Although not all project and Gem documentation may be included in O3DE Documentation, a reference to them can be listed in the O3DE Documentation. For example, PopocornFX's particle effects is listed in the "Rendering" page of the O3DE Documentation. Additionally, the D&C SIG will create two "external reference" pages: one in the Gem reference section of the User Guide, listing "External Gems and Templates", and another in the Tutorials & Examples section, listing "External Samples and Tutorials". Each page would contain a table with a row for each external reference that could include columns for a description, optional screenshot, attribution, and link to its source and documentation. Each reference page could also contain appropriate "user beware" banner notices informing the reader of the risk.
```

Reply:
+1 - This is described excellently. Thanks for the addition!